src/web/price_providers/dummy.ts

```diff
@@ -1,7 +1,7 @@
 import { PriceProvider, Price, Tokens } from "./interface"
 
 export class DummyPriceProvider implements PriceProvider {
-  async price(contractAddress: string, tokenId: number): Promise<Price> {
+  async price(): Promise<Price> {
```

Comment:
It should correspond to the interface, I don't think it is okay to remove this arguments

Reply:
It's actually allowed in case those parameters are not used. This doesn't break typing system and it's probably better than unused variables. Another option would be to make this `DummyPriceProvider` more verbose by actually starting to use the provided arguments.